src/app/asset-filters/asset-filters.component.ts

```diff
@@ -152,7 +152,7 @@ export class AssetFilters {
               }
               // Check if there are multiple selected Geo facets.
               if(paramName === 'geography') {
-                parsedParam = parsedParam.split(',')
+                parsedParam = parsedParam
               }
```

Comment:
It seems like this whole block could just go away instead? (`parsedParam = parsedParam` seems not to have any effect)

Reply:
Now that I have read your comment, I feel silly for not thinking about this earlier.

Reply:
@daneah change made in the commit `4e16fa5`.